README.md

```diff
@@ -140,6 +140,14 @@ Persisting will also speed up computation. This statement will persist an RDD in
 from pyspark import StorageLevel
 df.persist(storageLevel = StorageLevel(True, True, False, True, 1))
 ```	
+
```

Comment:
I like the text but don't know if the placement is good. This option applies not only to Python and we don't want to duplicate it again for the Scala example.

What if we push this further down into the `Configuration on Spark SQL temp table` section? We also have to add a new record to that table to mention the option there.